### PR TITLE
Update tutorial to work with launch site and controls update

### PR DESCRIPTION
**Pull request by ProbablePrime**
I noticed a few people in gitter trying to follow along with the tutorial and having trouble.

After helping them out I thought i'd update it to work with the launch version of the developer lab and the controls update.

To that end i've re-written quite considerable chunks of the tutorial. I've tried to keep it in line with Connor's original tone and method of writing.

Oh could you also advise me on how I might see a rendered output of the files? I noticed that my previous pull request made the `npm install` line from the start of the tutorial scroll horizontally on the actual docs website and would like to preview my changes as they'll look if they should be merged.

I've also made changes to the underlying code that the tutorial ends up producing to catch a few crashes which may confuse new users.

* I've added a callback to the robot handshake as that is required at the moment
* I've wrapped the mouse movement code in an if to catch blank joystick reports that are sent before anyone touches the joystick.

Please give it a read through and let me know if you'd like any changes.

### CODE REVIEW

File: tutorial.rst [hunk 1]
```diff
@@ -90,14 +90,23 @@ The most notable change is that, rather than trying to join a channel's chat, we
...
         var robot = new Tetris.Robot(details);
-        robot.handshake();
+        robot.handshake(function (err) {
+            if (err) {
+                console.log('There was a problem connecting to Tetris');
+                console.log(err);
+            } else {
+                console.log('Connected to tetris');
+            }
+        });
```
Comment (connor4312) [inline tutorial.rst]: Hm, I'd just have `if (err) throw new Error('Error connecting to Tetris', err);`. We want to keep this concise :smile:

**Comment by connor4312** [comment]
This looks good, thanks! I'm flying to Seattle tomorrow and will look at bringing these docs fully up-to-date on the plane.

@MCProhosting-Matt any copy changes you want to make.

**Comment by ProbablePrime** [comment]
Awesome, thanks for the feedback. I've adjusted the callback to that simple one liner.

**Comment by MCProhosting-Matt** [comment]
Apparently I can't edit this. It's fine, I'll get it merged and make some copy changes later. :)

**Comment by ProbablePrime** [comment]
:D Awesome!

**Pull request merged**